### DISCUSSION TITLE
Can the public mode prohibit any changes?

### DISCUSSION THREAD

**Discussion by LWBobo** [photoprism/photoprism#3914] [Q&A]
By setting PHOTOPRISM_AUTH_MODE to public, can I completely disable any Settings for the viewer? Actions such as creating new albums, categorizing photos, and adding labels to calendars are prohibited, only viewing is allowed, and no changes are allowed.

**Comment by LWBobo** [comment]
I know that I can disable the album function in the Settings file, but if I want to keep the album function, no one is allowed to do anything with the album in public mode: such as adding albums, adding pictures to the album.What Settings can I use to do this?

**Reply by adamzvolanek** [reply]
I'm not finding the Album checkbox in settings, can you point me to how I can disable album creation?

**Reply by adamzvolanek** [reply]
> I'm not finding the Album checkbox in settings, can you point me to how I can disable album creation?

Responding to my own thread with an answer should someone need it as well: https://github.com/photoprism/photoprism/discussions/4159#discussioncomment-9042240

**Comment by graciousgrey** [comment]
The public mode was developed for our public demo and to use PhotoPrism in a local network without authentication.

_PHOTOPRISM_AUTH_MODE: public is NOT recommended for instances installed on a server accessible from outside your home network._

https://docs.photoprism.app/getting-started/config-options/

If you want to give people read-only access you can use sharing links or the viewer user role:
- https://docs.photoprism.app/user-guide/share/
- https://docs.photoprism.app/user-guide/users/roles/#viewer
